libfreerdp/core/transport.c

```diff
@@ -750,20 +751,7 @@ int transport_check_fds(rdpTransport* transport)
 	if (!transport)
```

Comment:
I have to comment here as github don't allow me to add comment on hidden line.
We may need to also remove 'HANDLE event' above as it is never used now.